### ISSUE TITLE
[feature request] supporting .vue file

### ISSUE THREAD

**Issue by beeplin** [mrmlnc/vscode-csscomb#69]
Now csscomb supports <style> in html files and works amazingly fine. I wonder if it's feasible to make it working with .vue files too.

.vue file is basically the same with .html file. For csscomb, the only difference is that, .vue files may have attributes in <style> label: 

```vue
<template>
...
</template>
<script>
...
</script>
<style scoped lang="less"> <!-- HAS ATTRS HERE-->
...
</style>

**Comment by mrmlnc** [comment]
Very soon. For me: https://github.com/mrmlnc/vscode-csscomb/blob/master/src/services/comb.ts#L154

**Comment by mrmlnc** [comment]
Available in `vscode-csscomb@5.0.0`. Also, please, read [release notes](https://github.com/mrmlnc/vscode-csscomb/releases/tag/5.0.0).

**Comment by beeplin** [comment]
It works for <style> in html files, but not for .vue files. Not sure why~~
